ReactNativeClient/lib/models/Setting.js

```diff
@@ -407,6 +407,19 @@ class Setting extends BaseModel {
 			tagHeaderIsExpanded: { value: true, type: Setting.TYPE_BOOL, public: false, appTypes: ['desktop'] },
 			folderHeaderIsExpanded: { value: true, type: Setting.TYPE_BOOL, public: false, appTypes: ['desktop'] },
 			editor: { value: '', type: Setting.TYPE_STRING, subType: 'file_path_and_args', public: true, appTypes: ['cli', 'desktop'], label: () => _('Text editor command'), description: () => _('The editor command (may include arguments) that will be used to open a note. If none is provided it will try to auto-detect the default editor.') },
+			'export.pdfPageSize': { value: 'A4', type: Setting.TYPE_STRING, isEnum: true, public: true, appTypes: ['desktop'], label: () => _('Page size for PDF export'), options: () => {
+				return {
+					'A4': _('A4'),
+					'Letter': _('Letter'),
+				};
+			}},
+			'export.pdfPageOrientation': { value: false, type: Setting.TYPE_BOOL, isEnum: true, public: true, appTypes: ['desktop'], label: () => _('Page orientation for PDF export'), options: () => {
+				return {
+					false: _('Portrait'),
+					true: _('Landscape'),
+				};
+			}},
```

Comment:
I know there's only two values but I find it a bit confusing that it's true/false. Could you change it to a string enum with values "portrait" and "landscape"?

Then when calling printToPdf you can just do `landscape: Setting.value('export.pdfPageOrientation') === 'landscape'`

Reply:
Yes, will do, if you really think this is better. I'll be changing it in the next 30 minutes.

Reply:
Done.